src/VisualStudio/Core/Next/Remote/JsonRpcConnection.cs

```diff
@@ -5,64 +5,59 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Execution;
 using Microsoft.CodeAnalysis.Remote;
-using Roslyn.Utilities;
-using StreamJsonRpc;
-using Microsoft.CodeAnalysis.Internal.Log;
 
 namespace Microsoft.VisualStudio.LanguageServices.Remote
 {
-    internal class ServiceHubJsonRpcConnection : RemoteHostClient.Connection
+    internal class JsonRpcConnection : RemoteHostClient.Connection
     {
         // communication channel related to service information
-        private readonly ServiceJsonRpcClient _serviceClient;
+        private readonly ServiceJsonRpcEx _serviceRpc;
 
         // communication channel related to snapshot information
-        private readonly SnapshotJsonRpcClient _snapshotClient;
+        private readonly RemotableDataJsonRpcEx _remoteDataRpc;
```

Comment:
I'm not really getting the Ex naming.  Do we have a RemotableDataJsonRpc and a RemotableDataJsonRpcEx?